srcpkgs/python3-coverage/patches/no-versioned-coverage.patch

```diff
@@ -1,16 +0,0 @@
---- a/setup.py
```

Comment:
this patch is still needed: https://github.com/void-linux/void-packages/actions/runs/4400142292/jobs/7705163969?pr=42735#step:9:20

Reply:
The patch could be replaced with the equivalent:
```
post_install() {
       # remove versioned scripts (keep only /usr/bin/coverage)
       rm ${DESTDIR}/usr/bin/coverage3
       rm ${DESTDIR}/usr/bin/coverage-${py3_ver}
}
```